### ISSUE TITLE
Some operational amplifiers seem to have the power pins on every unit (visible)

### ISSUE THREAD

**Issue by poeschlr** [KiCad/kicad-symbols#412]
According to this forum post https://forum.kicad.info/t/filter-by-number-of-pins-broken/10076/5 this might make problems with the pin number filter. (It is also against what i remember we decided some time back)

**Comment by jkriege2** [comment]
yes, I think they haven't been updated yet ... mostly because there was no definite decision (if I remeber correctly) how to handle them with 1-4-unit variants)

**Comment by evanshultz** [comment]
Fixed with https://github.com/KiCad/kicad-symbols/pull/578.

However, at https://github.com/KiCad/kicad-symbols/pull/578#issuecomment-390287547 I opened a related discussion:
An option would be to put the triangle symbol on all units. Then it's obvious if many pins are on the power unit as they're not floating due to having some containing shape, and it still allows stacking the power unit on other units.
![image](https://user-images.githubusercontent.com/1936989/40246691-828b2104-5a7e-11e8-8f1a-f4e7fe7826d5.png)
